view/src/components/foodbanks.js

```diff
@@ -847,22 +851,7 @@ class Foodbank extends Component {
                             Loading Dock: {foodbank.loadingDock ? "yes" : "no"}
                           </Typography>
                         </Box>
-                        {/* --> this will be implemented when we get hours from Place API <--
-                        <Box p={3}>
-                          <Typography
-                            className={classes.pos}
-                            color="textSecondary"
-                          >
-                            Hours of Operation:
-                          </Typography>
-                          <Typography variant="body2" component="p">
-                            Monday-Friday: 9am - 5pm
-                            <br />
-                            Saturday: 10am - 4pm
-                            <br />
-                            Sunday: closed
-                          </Typography>
-                        </Box> */}
+                        {/* TODO(andrewhojel): do we include hours of foodbanks? */}
```

Comment:
Nit: I'd reword this TODO to be something like "Determine if we should include hours for foodbanks"